<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigTool</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Login</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_3username"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_2IP">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>192.168.1.147</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_IP">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_2port"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_4password"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4password">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2Port">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3username">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6ConnectedState">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>20</horstretch>
         <verstretch>9</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>11</width>
         <height>8</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>31</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditInfoOutput">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>511</width>
     <height>241</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;wget -P/opt http://192.168.15.72/C:/Users/86158/.ssh/id_rsa.pub&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2CleanOutput">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SearchIP">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewIP">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_allexe">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>execute</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewcmdlist">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addCmd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>31</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_5command"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2send">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>620</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtontest111">
      <property name="text">
       <string>test111</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtontest222">
      <property name="text">
       <string>test222</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtontest333">
      <property name="text">
       <string>test333</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>650</y>
     <width>216</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditAdmin"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAdmin">
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
